src/screens/VisualTests/VisualTests.tsx

```diff
@@ -8,8 +8,8 @@ import { Bar, Col, Row, Section, Sections, Text } from "../../components/layout"
 import { TooltipMenu } from "../../components/TooltipMenu";
 import { getFragment, graphql } from "../../gql";
 import {
-  BuildQuery,
-  BuildQueryVariables,
+  AddonVisualTestsBuildQuery as BuildQuery,
+  AddonVisualTestsBuildQueryVariables as BuildQueryVariables,
```

Comment:
Is the renaming necessary or should we just be renaming the types below?

Reply:
The latter I guess, I'll make the change.